### PR TITLE
Implementing OIDC sso having login radius as identity provider

### PR DESCRIPTION
**Pull request by SanjayV0**
#### Please update the following checklist as applicable. Otherwise, the maintainers may close your pull request.

- [x] You created a branch from `develop` branch.
- [x] You PR is raised on `develop` branch and not on`main`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/main/GUIDELINES.md).
- [x] You have read the [Contribution Guidelines](https://github.com/LoginRadius/engineering-portal/blob/main/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content elsewhere on the internet on platforms like Medium, Dev.to, etc.
- [x] The content is not copy-pasted from elsewhere on the internet.

### CODE REVIEW

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
+---
+title: "IMPLEMENTING OIDC SSO HAVING LOGIN RADIUS AS IDENTITY PROVIDER"
```
Comment (mohammed786) [inline content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:2]: there should not be a gap between LOGIN and RADIUS, its a name right

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+date: "2024-05-03"
+#coverImage: "IMPLEMENTING OIDC SSO HAVING LOGIN RADIUS AS IDENTITY PROVIDER.png"
```
Comment (mohammed786) [inline content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:4]: I think coverimage is required, try to create one and add,

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+Once the OIDC application is created, LoginRadius empowers administrators to tailor user claim fields effortlessly. These customizable user claims can be finely tuned through the platform's user-friendly interface. Subsequently, these specified claims are seamlessly integrated into the token, enabling streamlined extraction and utilization within the application ecosystem.
+
+In essence, LoginRadius not only facilitates the setup of OIDC applications but also enhances customization capabilities through its intuitive interface. This ensures efficient management of user claims, ultimately contributing to a more personalized and secure authentication experience.
+
+
+After setting up the oidc app from the login radius dashboard we will use the go -oidc library to further configure our provider and configure the oidc connect.
+
+## Setting up OIDC Application
+
+First go to Admin Dashboard and Navigate to
```
Comment (mohammed786) [inline content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:29]: You can add a link of admin console for better context

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+provider, err := oidc.NewProvider(ctx, "https://api.loginradius.com/{oidcappname}")
+if err != nil {
+    // handle error
+}
+
+// Configure an OpenID Connect aware OAuth2 client.
+oauth2Config := oauth2.Config{
+    ClientID:     your-oidc-clientID,
+    ClientSecret: your-oidc-clientSecret
+    RedirectURL:  redirectURL,
```
Comment (mohammed786) [inline content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:98]: I think you should cover the details of how to get this client ID and Client's secret

**Comment by lr-hemant** [comment]
@SanjayV0 Grammar check should be performed before submitting the MR.
cc: @mohammed786

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md (hunk 1 above)
Comment (SanjayV0) [reply content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:2]: okay changed

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md (hunk 2 above)
Comment (SanjayV0) [reply content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:4]: ya will add one

File: content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md (hunk 3 above)
Comment (SanjayV0) [reply content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md:29]: Sure will add it .

**Review by mohammed786 — APPROVED** [review]

**Comment by SanjayV0** [comment]
Review needed @raghunath-r-a

**Pull request merged**